src/transpiler.ts

```diff
@@ -1,22 +1,25 @@
 import * as fs from 'fs';
 import { cwd } from 'process';
-import * as ts from 'typescript';
+// Only for types; take care never to use ts_ in expressions, only in type annotations
+import * as ts_ from 'typescript';
```

Comment:
let's call it ts_types or something a little more meaningful

Reply:
Or instead of using typeof everywhere - perhaps export a type that's defined as

Reply:
Unfortunately type aliases cannot be used with namespaces, so `type TsTypes = typeof ts´ is rejected by the compiler.  I'll use `ts_types`.